replication-plugin.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<!--
+You can configure one or more peer servers here.
+Each server configuration should be placed within <servers></servers> tag.
+Server configuration example:
+<server>
+    <url>http://localhost:8083/nexus</url>
+    <user>admin</user>
+    <password>admin123</password>
+</server>
+
+User and password are optional. If there is neither login nor password (or login is empty) server will use anonymous user.
+You could also define myUrl attribute for <configurations> tag that defines URL of the Nexus instance.
```

Comment:
I just thought.. It will probably be useful to explain _why_ we need this _myUrl_ and how it's used:
- At least it will be clear for users
- Since Nexus may have different URLs, users may not understand what particular URL is expected here. So the doc will serve as a hint for them.